src/components/ApplicationTimer.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react'
+import { calculateTimeLeft, formatDoubleDigitTime } from '../utils/timer'
+import styles from '../css/CountdownTimer.module.css'
+
+const getTime = dueDate => {
+  const timeLeft = calculateTimeLeft(dueDate)
+
+  if (timeLeft) {
+    timeLeft.hours = formatDoubleDigitTime(timeLeft.hours)
```

Comment:
have you read on [String.prototype.padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)? you can use
```timeLeft.hours.toString().padStart(2, '0')```